project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=1.9.7
+sbt.version=1.9.9
```

Comment:
Is it strictly necessary to update the SBT version in this PR?

Reply:
Yes, it is a good practice to upgrade to latest version of `scala`, `sbt` and sbt plugins when upgrading project.